scripts/unicode_lister.rb

```diff
@@ -163,18 +164,18 @@ def calculate_ranges(matching_codes)
     end
   end
     .map { |range| range.size == 1 ? range.first : range}
-    .join(", ")
 end
 
 count = 0
-File.open(OutputFilename, 'w') do |f|
+store = PStore.new("db/#{RegexpExamples::UnicodeCharRanges::STORE_FILENAME}")
+store.transaction do
   NamedGroups.each do |name|
   count += 1
     matching_codes = (0..55295).lazy.select { |x| /\p{#{name}}/ =~ eval("?\\u{#{x.to_s(16)}}") }.first(128)
-    f.puts "'#{name.downcase}' => ranges_to_unicode(#{calculate_ranges(matching_codes)}),"
+    store[name.downcase] = calculate_ranges(matching_codes)
```

Comment:
.... Or in other word (as per my above comment), I think we should either be using a human-readable YAML::Store (which might not be a bad idea?), or just do:

     store[name.downcase] = (0..55295).lazy.select { |x| /\p{#{name}}/ =~ eval("?\\u{#{x.to_s(16)}}") }.first(128)